### PR TITLE
Require an auth token for api calls.

### PR DESCRIPTION
**Pull request by zenhack**
The CI for this will fail until https://github.com/CCI-MOC/obmd/pull/39 lands; same reason as before. Built atop #22.

### CODE REVIEW

File: cmd/hil-vpnd/api_test.go [hunk 1]
```diff
@@ -77,6 +77,26 @@ func initTestServer(ops *MockPrivOps) *httptest.Server {
 	return server
 }
 
+// Test that we reject unathenticated API calls.
```
Comment (ianballou) [inline cmd/hil-vpnd/api_test.go]: unauthenticated *

**Review by ianballou — APPROVED** [review]
Just a typo, but besides that everything looks good!

File: cmd/hil-vpnd/api_test.go (hunk 1 above)
Comment (zenhack) [reply cmd/hil-vpnd/api_test.go]: Fixed. Also caught another typo while i was in there.

**Review by naved001 — APPROVED** [review]

**Comment by naved001** [comment]
should we rebase this before merging?

**Comment by zenhack** [comment]
I just did the rebase & merge via GitHub's UI; the code should be exactly the same, it's just missing the merge commit, so it didn't seem important to push and have Travis test again.

**Pull request merged**